frontend/src/pages/__snapshots__/PipelineDetails.test.tsx.snap

```diff
@@ -64,6 +64,7 @@ exports[`PipelineDetails closes side panel when close button is clicked 1`] = `
             <div>
               test-pipeline-id
             </div>
+            0
```

Comment:
I'm curious why this is 0?

Reply:
My guess is: this place is the version selector; and we show default version (index 0) when we open pipeline details by clicking the pipeline (instead of clicking a particular pipeline version)

Reply:
done